src/components/header/Header.jsx

```diff
@@ -15,7 +18,7 @@ export default function Header() {
           <p className="whiteLine mt-4"></p>
         </div>
         <div className="headerTextContent">
-          <h1 className="headerTitle">We Care About Your Health</h1>
+          <h1 className="headerTitle">{t("We Care About Your Health")}</h1>
```

Comment:
this doesnt seem right, the translations are usually short keys, not full sentences?